src/Views/Private/Chat/components/Messages/Messages.vue

```diff
@@ -18,7 +23,11 @@ export default defineComponent({
 	},
 	computed: {
 		isSender() {
-			return this.message.sender === 13
+			if (this.message.sender === 13) {
+				return true
+			} else {
+				return false
+			}
```

Comment:
Suggestion: the operator === already returns true or false, this whole function can be replaced by 
```suggestion
			return this.message.sender === 13
```